<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PDF合并&amp;拆分</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>r5.ico</normaloff>r5.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="filepath">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请选择要分割的单个PDF文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分割范围用“-”连接，多个范围用空格分开。比如（1-1 2-5 6-10 11-20）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>分割方式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_split">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始分割</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../j12.ico</normaloff>../j12.ico</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_split_std">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>1-1 2-5 6-10 11-20</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_log">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>441</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>记录分割的过程</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>文件路径</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_file">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../u17.ico</normaloff>../u17.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>多个文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filepath_merge">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请选择要合并的多个PDF文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_file_merge">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择多个文件</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../u17.ico</normaloff>../u17.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_merge">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>合并PDF</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../j13.ico</normaloff>../j13.ico</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
